src/renderer/pages/Staking/Overview/components/NominatorsList/NominatorsList.tsx

```diff
@@ -68,36 +33,52 @@ export const NominatorsList = ({
   onToggleNominator,
 }: Props) => {
   const { t } = useI18n();
+  const { getNextUnstakingEra, hasRedeem } = useStakingData();
+
+  const getUnstakeBadge = (stake: NominatorInfo) =>
+    getNextUnstakingEra(stake.unlocking, era) && (
+      <Tooltip offsetPx={-65} content={<Trans t={t} i18nKey="staking.tooltips.unstakeDescription" />}>
+        <div className="flex gap-x-1 items-center rounded-md bg-badge-background px-2 py-0.5">
+          <Icon name="unstake" className="text-icon-accent" size={14} />
+          <HelpText className="text-icon-accent">
+            <TimeToEra className="my-1" api={api} era={getNextUnstakingEra(stake.unlocking, era)} />
+          </HelpText>
+        </div>
+      </Tooltip>
+    );
 
-  const activeWallet = useUnit(walletModel.$activeWallet);
-
-  const getExplorers = (address: Address, stash?: Address, explorers: Explorer[] = []) => {
-    const explorersContent = explorers.map((explorer) => ({
-      id: explorer.name,
-      value: <ExplorerLink name={explorer.name} href={getAccountExplorer(explorer, { address })} />,
-    }));
-
-    if (!stash) return [{ items: explorersContent }];
-
-    const validatorsButton = (
-      <button
-        type="button"
-        className="flex items-center gap-x-2 px-2 w-full h-full"
-        onClick={() => onCheckValidators(stash)}
-      >
-        <Icon name="viewValidators" size={16} />
-        <FootnoteText as="span" className="text-text-primary">
-          {t('staking.overview.viewValidatorsOption')}
-        </FootnoteText>
-      </button>
+  const getRedeemBadge = (stake: NominatorInfo) =>
+    hasRedeem(stake.unlocking, era) && (
+      <Tooltip offsetPx={-65} content={<Trans t={t} i18nKey="staking.tooltips.redeemDescription" />}>
+        <div className="flex gap-x-1 items-center rounded-md bg-positive-background text-text-positive px-2 py-0.5">
+          <Icon name="redeem" className="text-text-positive" size={14} />
+          <HelpText className="text-text-positive">{t('staking.tooltips.redeemTitle')}</HelpText>
+        </div>
+      </Tooltip>
     );
 
-    return [{ items: [{ id: '0', value: validatorsButton }] }, { items: explorersContent }];
-  };
+  const getContent = (stake: NominatorInfo) => (
+    <>
+      {(stake.account as Account).type === 'shard' ? (
```

Comment:
For such conditions we must use type guard functions `accountUtils.isShardAccount`